app/controllers/DataSetController.scala

```diff
@@ -72,6 +75,21 @@ class DataSetController @Inject()(val messagesApi: MessagesApi) extends Controll
     }
   }
 
+  def addForeignDataStoreAndDataSet(url: String, dataStoreName: String, dataSetName: String) = SecuredAction.async { implicit request =>
```

Comment:
should be for admins only (we might grant it to team managers as well later on, but surely not to everybody)

Reply:
thanks, I see you added it :)